src/stockTracker.js

```diff
@@ -10,6 +10,83 @@ import {
 
 const ComponentTitle = ({ title }) => <h3 className="component-title">{title}</h3>
 
+const AdaptiveLogo = () => (
```

Comment:
Perhaps put the adaptiveLogo in another file. It will make the module look cleaner :-D